pom.xml

```diff
@@ -35,6 +36,8 @@
     </licenses>
 
     <modules>
+        <module>http-delegation</module>
```

Comment:
Since the two Gradle projects now have a POM with packing type pom, only the license-maven-plugin is executed.